examples/spisd/device.c

```diff
@@ -36,25 +36,6 @@
 #define SIGF_OP_REQUEST (1 << SIGB_OP_REQUEST)
 #define SIGF_OP_TIMER (1 << SIGB_TIMER)
 
-#ifndef TD_GETGEOMETRY
```

Comment:
Are you assuming that I should install some additional code (NDK 3.2?) for this to work?

Currently I have vbcc installed using the installer here, https://eab.abime.net/showthread.php?t=83113, and trying to compile using the `examples/spisd/build.bat` script fails regardless if I'm using NDK 13 or NDK 39.

Reply:
Yes install and use NDK 3.2, it is available on aminet.

The 3.2 NDK is very convenient for driver development as it has all defines and structures needed, even both NSD and TD64, so no additional includes need to be added or pasted in the code.